Makefile

```diff
@@ -35,3 +36,20 @@ image-ose-deployer: image-ose-cli
 $(call build-image,ose-recycler,./images/recycler/Dockerfile.rhel,.)
 image-ose-recycler: image-ose-cli
 
+update: update-generated-completions
+.PHONY: update
+
+verify: build verify-cli-conventions verify-generated-completions
```

Comment:
verify doesn't depend on build

Reply:
> verify doesn't depend on build

it requires `oc`

Reply:
not the `verify` target, the `verify-*` might